tests/v3_api/test_auth.py

```diff
@@ -576,11 +624,13 @@ def enable_openldap_nestedgroup(username, token, expected_status=200):
     print "Enable openLdap nestedgroup request for " + username + " " + str(expected_status)
 
 
-def enable_ad(username, token, expected_status=200):
+def enable_ad(username, usersearchfilter, groupsearchfilter, token, expected_status=200):
```

Comment:
@carolynhu Could you define a default value of "" for usersearchfilter and groupsearchfilter parameters and have them added after token ? This way we dont have to pass  empty string values in cases were these fields need not be set while enabling auth.